src/content-scripts/components/AICodeRefactor/ChangeSuggestorButton.ts

```diff
@@ -0,0 +1,81 @@
+import { createHtmlElement } from "../../../utils/createHtmlElement";
+import openSaucedLogoIcon from "../../../assets/opensauced-icon.svg";
+import { GITHUB_PR_SUGGESTION_TEXT_AREA_SELECTOR, OPEN_AI_COMPLETION_MODEL_NAME, SUPABASE_LOGIN_URL } from "../../../constants";
+import { generateCodeSuggestion } from "../../../utils/aiprdescription/openai";
+import { isContextWithinBounds } from "../../../utils/fetchGithubAPIData";
+import { insertAtCursorFromStream } from "../../../utils/aiprdescription/cursorPositionInsert";
+import { getAIDescriptionConfig } from "../../../utils/aiprdescription/descriptionconfig";
+import { isLoggedIn } from "../../../utils/checkAuthentication";
+
+export const ChangeSuggestorButton = (commentNode: HTMLElement) => {
+    const changeSuggestoreButton = createHtmlElement("a", {
+      innerHTML: `<span id="ai-change-gen" class="toolbar-item btn-octicon">
+      <img class="octicon octicon-heading" height="16px" width="16px" id="ai-description-button-logo" src=${chrome.runtime.getURL(openSaucedLogoIcon)}>
+      </span>
+      <tool-tip for="ai-change-gen" class="sr-only" role="tooltip">Get Refactor Suggestions</tool-tip>`,
+      onclick: async () => handleSubmit(commentNode),
+    });
+
+    return changeSuggestoreButton;
+  };
```

Comment:
```suggestion
export const ChangeSuggestorButton = (commentNode: HTMLElement) => {
  const changeSuggestorButton = createHtmlElement("a", {
    innerHTML: `<span id="ai-change-gen" class="toolbar-item btn-octicon">
      <img class="octicon octicon-heading" height="16px" width="16px" id="ai-description-button-logo" src=${chrome.runtime.getURL(openSaucedLogoIcon)}>
    </span>
    <tool-tip for="ai-change-gen" class="sr-only" role="tooltip">Get Refactor Suggestions</tool-tip>`,
    onclick: () => handleSubmit(commentNode),
  });
  return changeSuggestorButton;
};
```
- Removed unnecessary async from onclick function since there are no asynchronous operations inside the function call.
- Fixed typo in function name "changeSuggestoreButton" to "changeSuggestorButton".
Description generated using [OpenSauced](https://opensauced.ai/).

Reply:
nice 😅